packages/sample-app/sample_app/strands_example.py

```
@@ -0,0 +1,410 @@
+                     "- Budget estimation and financial planning\n" +
+                     "- Optimizing travel schedules and logistics\n" +
+                     "- Balancing activities with traveler preferences\n" +
+                     "Create well-structured, realistic travel plans."
+    )
+    
+    # Booking Agent - Specialized in reservations
+    booking_agent = Agent(
+        model=model,
+        tools=[search_flights, search_hotels, get_memory_context],
+        system_prompt="You are a Travel Booking Specialist. Your expertise includes:\n" +
+                     "- Finding the best flight deals and options\n" +
```

Flake8 (7.2.0)

[error] continuation line unaligned for hanging indent

(E131)